apps/Sphero SDK/description

```diff
@@ -0,0 +1,17 @@
+(BEFORE INSTALLATION: ENABLE SERIAL PORT, THEN DISABLE SERIAL CONSOLE IN RASPBERRY PI CONFIG)
```

Comment:
First line of the description will be the tooltip. I think this can be improved.

Reply:
Should I change it to make more sense on how to activate those two?

Reply:
Or, explain its purpose?